apps/dsp-app/src/app/project/ontology/resource-class-info/resource-class-property-info/resource-class-property-info.component.ts

```diff
@@ -143,16 +145,19 @@ export class ResourceClassPropertyInfoComponent implements OnChanges, AfterConte
     }
 
     // get current ontology lists to get linked list information
-    const currentOntologyLists = this._store.selectSnapshot(ListsSelectors.listsInProject);
-    if (currentOntologyLists && this.propDef.objectType === Constants.ListValue) {
-      // this property is a list property
-      const re = /\<([^)]+)\>/;
-      const listIri = this.propDef.guiAttributes[0].match(re)[1];
-      const listUrl = `/project/${this.projectUuid}/list/${listIri.split('/').pop()}`;
-      const list = currentOntologyLists.find(i => i.id === listIri);
-      this.propAttribute = `<a href="${listUrl}">${list.labels[0].value}</a>`;
-      this.propAttributeComment = list.comments.length ? list.comments[0].value : null;
-    }
+    this.subscription = this._store.select(ListsSelectors.listsInProject).subscribe(currentOntologyLists => {
+      if (currentOntologyLists && this.propDef.objectType === Constants.ListValue) {
+        // this property is a list property
+        const re = /\<([^)]+)\>/;
+        const listIri = this.propDef.guiAttributes[0].match(re)[1];
+        const listUrl = `/project/${this.projectUuid}/list/${listIri.split('/').pop()}`;
+        const list = currentOntologyLists.find(i => i.id === listIri);
+        if (list) {
+          this.propAttribute = `<a href="${listUrl}">${list.labels[0].value}</a>`;
```

Comment:
consider creating observable property and moving this logic out from ngafterinit, if possible